website/config.yaml

```diff
@@ -66,8 +66,9 @@ params:
         url: 'https://slack.k8s.io/'
         icon: fab fa-slack
         desc: 'Chat with us on Slack in the #aws-provider channel'
-  latest_release_version: v0.15.0
+  latest_release_version: v0.16.0
   versions:
+    - "v0.16.0"
     - "v0.15.0"
     - "v0.14.0"
     - "v0.13.2"
```

Comment:
Can you also remove v0.12.1? We've only been keeping 4 versions in the drop down.